Nodejs/Product/Nodejs/Project/NodejsProjectLauncher.cs

```diff
@@ -468,6 +464,29 @@ private bool ShouldStartBrowser()
             return true;
         }
 
+        private string GetBrowserPath()
+        {
+            ThreadHelper.ThrowIfNotOnUIThread();
+
+            var uiShellOpenDocument = (IVsUIShellOpenDocument)ServiceProvider.GlobalProvider.GetService(typeof(SVsUIShellOpenDocument));
+            uiShellOpenDocument.GetFirstDefaultPreviewer(out var defaultBrowserPath, out _, out _);
```

Comment:
The docs state this method: "Returns S_OK if a default previewer is defined. Returns S_FALSE if there are no default previewers". You don't seem to allow for the latter case here.

Reply:
This is as intended as we later check if the browserPath exists.